### ISSUE TITLE
Support generation of multiple schemas within it's own namespace

### ISSUE THREAD

**Issue by x80486** [bjornvester/xjc-gradle-plugin#4]
Would be nice if there is a way to configure settings per schema, as well as for a group of them.

Something along these lines:

```groovy
xjc {
  schemas {
    firstSet {
      defaultPackage = "tld.domain.first"
      markGenerated = true
      xsdDir = project.file("${project.rootDir}/assets/schemas/first/")
    }
    secondSet {
      defaultPackage = "tld.domain.second"
      markGenerated = true
      xsdDir = project.file("${project.rootDir}/assets/schemas/second/")
    }
  } 
}
```
In that way, it would be possible to use **Different package names/identifiers for a given "set"**. I think that would be a key piece for this plugin.

**Comment by bjornvester** [comment]
Yeah, something along those lines could be useful. I will see if I can whip something up when I get the time.

**Comment by StephanBeutel** [comment]
Would be a great feature. Other plugins available which support this feature, do not support Java 11

**Comment by Eaelenthil** [comment]
I suppose this can be achieved by using the task directly and defining multiple tasks, if I'm not mistaken. I've seen mention that this will be documented at a later date in the roadmap. I've attempted to configure multiple tasks in this way, but no luck so far. Could we get an example or a hint how we might go about using the task directly until this feature is implemented?

**Comment by bjornvester** [comment]
Yes, sorry I've been away for some time. Too busy with... life :)

It can be done with multiple tasks of type Xjc, and this was what I originally envisioned would be the "right" way to do it. But it turns out you need quite a lot of configuration to make it work, and that is just not very user friendly. So I think it might make more sense to do what this issue suggests and make it configurable through the extension. I am back now and will make a solution to this soon.

**Comment by Eaelenthil** [comment]
That would be much appreciated. However, I have one question. How would episode files work with this feature? Would there be a single episode file for all per schema configs or would there be per schema episode files?

**Comment by hortegag91** [comment]
Is there near a new release with this changes? It would be great.
